lib/registration.pm

```diff
@@ -562,7 +562,7 @@ sub registration_bootloader_params {
 }
 
 sub yast_scc_registration {
-    type_string "yast2 scc; echo sccreg-done-\$?- > /dev/$serialdev\n";
+    my $module = y2logsstep::yast2_console_exec(yast2_module => 'scc');
```

Comment:
Value  for key `yast2_module` and variable  `$module` seem to contain the same info . Do you think we could remove this return value, call it just `module => 'scc') as the name of the function is already referring to some YaST and concatenate in place: `yast2-' . $module . '-status`? We could call `module_status` but not sure if it is really a status as I think we always check `-0`. WDYT?

Reply:
ahm! I just realized this was already discussed. Anyway that would be my suggestion if possible to remove the return, if you prefer not to, I would opt for `module_name` instead of module as it was confused for me to know what was that variable.

Reply:
``$module`` stores the value of string which should appear with the return code on the serial console. To make it easy for the test developer so he does not need to handle it by himself. Regarding the return code, I rather leave it for the test developer, since it is common that utilities can return other codes than 0 which might be useful at some point. I am not really sure how it is with yast2 modules since the documentation does not go to details about the return codes

Reply:
from that point of view totally make sense, thanks for the clarification.

Reply:
alright, so I will change the labels and afterward, we can merge.